package.json

```diff
@@ -25,13 +25,13 @@
     "lint": "ng lint",
     "e2e": "ng e2e",
     "prebuild": "rimraf dist && yarn build:i18n && yarn build:browserslist",
-    "build": "yarn build:prod",
+    "build": "ng build --prod --localize",
     "postbuild": "ts-node -P scripts/tsconfig.json scripts/postbuild.ts --env production",
     "build:i18n:clone": "ts-node -P scripts/tsconfig.json scripts/translate-file-clone.prebuild.ts",
     "build:i18n:extract": "ng xi18n  --i18n-format=xlf2 --output-path locale",
     "build:i18n:generate": "ts-node -P scripts/tsconfig.json scripts/translate-file-generator.prebuild.ts",
     "build:i18n": "yarn build:i18n:extract && yarn build:i18n:generate",
-    "build:prod": "yarn prebuild && ng build --prod --localize && yarn postbuild",
+    "build:prod": "yarn build",
```

Comment:
@leomendoza123 I think we should leave it as it was before to run prebuild and postbuild as well or otherwise remove them also from qa and sandbox

Reply:
@DanielPalafox Good observation. Npm by default runs prebuild and post-build when you run `npm run build`. So With this I am just simplifying this script